### ISSUE TITLE
IEV data anomaly (20201217): Replacing 00a0 with normal space

### ISSUE THREAD

**Issue by ronaldtse** [glossarist/iev-data#50]
`\u00a0` are used in multiple SOURCE fields as a single whitespace.

Including:
```
Processing term 114-02-03 (eng)... [relaton-iec] ("IEC 60050-151") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-151: 2001. The code must be exactly like it is on the standards website.
Processing term 114-03-05 (eng)... [relaton-iec] ("IEC 60050-482") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-482. The code must be exactly like it is on the standards website.
Processing term 121-11-03 (eng)... 705-03‑01 MOD
Processing term 121-11-03 (fra)... 705-03‑01 MOD
Processing term 121-11-47 (eng)... 221-01‑03 MOD
Processing term 121-11-47 (fra)... 221-01‑03 MOD
Processing term 121-11-48 (eng)... 221-01‑02 MOD
Processing term 121-11-48 (fra)... 221-01‑02 MOD
Processing term 121-12-29 (eng)... 221-03‑ MOD
Processing term 121-12-29 (fra)... 221-03‑ MOD
Processing term 121-12-54 (eng)... 221-02‑44 MOD
Processing term 121-12-54 (fra)... 221-02‑44 MOD
Processing term 121-12-64 (eng)... 221-02‑38 MOD
Processing term 121-12-64 (fra)... 221-02‑38 MOD
Processing term 121-12-65 (eng)... 221-02‑39 MOD
Processing term 121-12-65 (fra)... 221-02‑39 MOD
Processing term 121-12-66 (eng)... 221-02‑40 MOD
Processing term 121-12-66 (fra)... 221-02‑40 MOD
Processing term 121-12-68 (eng)... 221-02‑35 MOD
Processing term 121-12-68 (fra)... 221-02‑35 MOD
Processing term 121-13-11 (eng)... 212-01‑33 MOD
Processing term 121-13-11 (fra)... 212-01‑33 MOD
Processing term 131-14-37 (eng)... 726-07-07 MOD
Processing term 131-14-37 (fra)... 726-07-07 MOD
Processing term 151-13-41 (eng)... 411-03-01 MOD
Processing term 151-13-41 (fra)... 411-03-01 MOD
Processing term 151-13-49 (eng)... 351-18-46 MOD
Processing term 151-13-49 (fra)... 351-18-46 MOD
Processing term 151-13-74 (eng)... 441-16-49 MOD
Processing term 151-13-74 (fra)... 441-16-49 MOD
Processing term 151-15-03 (eng)... 601-01-26 MOD
Processing term 151-15-03 (fra)... 601-01-26 MOD
Processing term 151-15-05 (eng)... 601-01-27 MOD
Processing term 151-15-05 (fra)... 601-01-27 MOD
Processing term 151-15-35 (eng)... 212-01-01 MOD
Processing term 151-15-35 (fra)... 212-01-01 MOD
Processing term 151-16-15 (eng)... [relaton-iso] ("ISO/IEC Guide 2 ") fetching...
[relaton-iso] WARNING: no match found online for ISO/IEC Guide 2 . The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO/IEC Guide 2  (all parts)".
Processing term 151-16-15 (fra)... [relaton-iso] ("ISO/IEC Guide 2 ") fetching...
[relaton-iso] WARNING: no match found online for ISO/IEC Guide 2 . The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO/IEC Guide 2  (all parts)".
Processing term 151-16-17 (eng)... 411-53-02 MOD
Processing term 151-16-17 (fra)... 411-53-02 MOD
Processing term 151-16-18 (fra)... [relaton-iso] ("ISO 3534-1") fetching...
[relaton-iso] WARNING: no match found online for ISO 3534-1. The code must be exactly like it is on the standards website.
Processing term 151-16-32 (eng)... 551-19-02 MOD
Processing term 151-16-32 (fra)... 551-19-02 MOD
Processing term 151-16-38 (eng)... 486-01-20/21 MOD
Processing term 151-16-38 (fra)... 486-01-20/21 MOD
Processing term 192-01-17 (eng)... [relaton-iso] ("ISO 9000:2005") fetching...
[relaton-iso] WARNING: no match found online for ISO 9000:2005. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO 9000:2005 (all parts)".
Processing term 192-10-17 (eng)... [relaton-iso] ("ISO/IEC/IEEE 24765:2010") fetching...
[relaton-iso] WARNING: no match found online for ISO/IEC/IEEE 24765:2010. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO/IEC/IEEE 24765:2010 (all parts)".
Processing term 192-13-01 (eng)... [relaton-iec] ("IEC 60050-311") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-311:2001. The code must be exactly like it is on the standards website.
Processing term 192-13-03 (eng)... [relaton-iec] ("IEC 60050-103") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-103:2009. The code must be exactly like it is on the standards website.
Processing term 311-03-08 (eng)... &ne: VIM 
Processing term 311-03-08 (fra)... &ne: VIM 
Processing term 311-03-17 (eng)... &asymp: VIM
Processing term 311-03-17 (fra)... &asymp: VIM
Processing term 311-04-01 (fra)... &asymp: VIM
Processing term 311-06-06 (eng)... &asymp: VIM
Processing term 311-06-06 (fra)... &asymp: VIM
Processing term 311-06-12 (eng)... &asymp: VIM
Processing term 311-06-12 (fra)... &asymp: VIM
Processing term 312-02-09 (eng)... &asymp: VIM
Processing term 312-02-09 (fra)... &asymp: VIM
Processing term 312-02-10 (eng)... &asymp: VIM
Processing term 312-02-10 (fra)... &asymp: VIM
Processing term 312-02-11 (eng)... &asymp: VIM
Processing term 312-02-11 (fra)... &asymp: VIM
Processing term 314-01-02 (fra)... &asymp: VIM
Processing term 314-01-03 (fra)... &asymp: VIM
Processing term 314-01-06 (eng)... &asymp: VIM
Processing term 314-01-06 (fra)... &asymp: VIM
Processing term 314-01-07 (eng)... &asymp: VIM
Processing term 314-01-07 (fra)... &asymp: VIM
Processing term 314-01-10 (eng)... &asymp: VIM
Processing term 314-01-10 (fra)... &asymp: VIM
Processing term 395-01-03 (eng)... [relaton-iec] ("IEC 60050-111") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-111:1996. The code must be exactly like it is on the standards website.
Processing term 395-01-08 (eng)... [relaton-iso] ("ISO 921:1997") fetching...
[relaton-iso] WARNING: no match found online for ISO 921:1997. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO 921:1997 (all parts)".
Processing term 395-01-34 (eng)... [relaton-iec] ("IEC 60050-881") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-881:1983. The code must be exactly like it is on the standards website.
Processing term 395-02-24 (eng)... [relaton-iec] ("IEC 60050-113") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-113:1996. The code must be exactly like it is on the standards website.
Processing term 395-02-33 (eng)... [relaton-iec] ("IEC 60050-531") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-531:1974. The code must be exactly like it is on the standards website.
Processing term 395-03-113 (eng)... [relaton-iec] ("IEC 61577-1") fetching...
[relaton-iec] WARNING: no match found online for IEC 61577-1:2006. The code must be exactly like it is on the standards website.
Processing term 395-05-43 (eng)... ICRU Report 39
Processing term 395-05-43 (fra)... ICRU Report 39
Processing term 395-07-120 (eng)... [relaton-iec] ("IEC 61504") fetching...
[relaton-iec] WARNING: no match found online for IEC 61504: 2000. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 61504 (all parts)".
Processing term 395-07-131 (eng)... [relaton-iec] ("IEC 61513") fetching...
[relaton-iec] WARNING: no match found online for IEC 61513:2011. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 61513 (all parts)".
Processing term 395-07-133 (eng)... [relaton-iec] ("IEC 61508-4") fetching...
[relaton-iec] WARNING: no match found online for IEC 61508-4:2010. The code must be exactly like it is on the standards website.
Processing term 395-07-134 (eng)... [relaton-iec] ("IEC 62303") fetching...
[relaton-iec] WARNING: no match found online for IEC 62303:2008. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 62303 (all parts)".
Processing term 395-07-47 (eng)... [relaton-iec] ("IEC 60880") fetching...
[relaton-iec] WARNING: no match found online for IEC 60880:2006. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 60880 (all parts)".
Processing term 395-08-13 (eng)... IAEA 4
Processing term 395-08-13 (fra)... AIEA 4
Processing term 415-02-10 (eng)... 191-05‑20 MOD
Processing term 415-02-10 (fra)... 191-05‑20 MOD
Processing term 415-03-11 (fra)... voir 415-03‑10
Processing term 415-03-13 (eng)... [relaton-iec] ("IEC 61400-1") fetching...
[relaton-iec] WARNING: no match found online for IEC 61400-1. The code must be exactly like it is on the standards website.
Processing term 442-01-46 (eng)... 441-14-01 MOD
Processing term 442-01-46 (fra)... 441-14-01 MOD
Processing term 442-02-34 (eng)... 826-06-04 MOD
Processing term 442-02-34 (fra)... 826-06-04 MOD
Processing term 445-04-03 (eng)... [relaton-iec] ("IEC 60947-1") fetching...
[relaton-iec] WARNING: no match found online for IEC 60947-1. The code must be exactly like it is on the standards website.
Processing term 651-26-06 (eng)... [relaton-iso] ("ISO/IEC Guide 51:1999") fetching...
[relaton-iso] WARNING: no match found online for ISO/IEC Guide 51:1999. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO/IEC Guide 51:1999 (all parts)".
Processing term 651-26-11 (eng)... [relaton-iec] ("IEC 60050-195") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-195:1998. The code must be exactly like it is on the standards website.
Processing term 802-01-06 (eng)... definition  of IEC 62127-1
Processing term 802-01-06 (fra)... définition  de la IEC 62127-1
Processing term 802-01-08 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-08 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-10 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-11 (eng)... definition  of IEC 62127-1
Processing term 802-01-11 (fra)... définition  de la IEC 62127-1
Processing term 802-01-12 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-12 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-13 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-13 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-14 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-14 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-15 (eng)... definition  of IEC 62127-1
Processing term 802-01-15 (fra)... définition  de la IEC 62127-1
Processing term 802-01-16 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-16 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-17 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-17 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-19 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-19 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-20 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-20 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-21 (fra)... définition  de la IEC 62127-1 MOD
Processing term 802-01-24 (eng)... definition  of IEC 62127-1
Processing term 802-01-24 (fra)... définition  de la IEC 62127-1
Processing term 802-01-25 (eng)... definition  of IEC 62127-1 MOD
Processing term 802-01-25 (fra)... définition  de la IEC 62127-1 MOD
Processing term 815-10-01 (eng)... [relaton-iec] ("IEC 60050-121") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-121:1998. The code must be exactly like it is on the standards website.
Processing term 815-13-05 (eng)... [relaton-iec] ("IEC 60050-461") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-461:2008. The code must be exactly like it is on the standards website.
Processing term 815-17-19 (eng)... [relaton-iec] ("IEC 60050-415") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-415:1999. The code must be exactly like it is on the standards website.
Processing term 841-27-03 (eng)... 121-13‑19 MOD
Processing term 841-27-03 (fra)... 121-13‑19 MOD
Processing term 841-28-31 (eng)... 411-53‑09 MOD
Processing term 841-28-31 (fra)... 411-53‑09 MOD
Processing term 841-28-33 (eng)... see CISPR 11
Processing term 841-28-33 (fra)... voir CISPR 11
Processing term 841-32-07 (eng)... 731-06-01 MOD
Processing term 841-32-07 (fra)... 731-06-01 MOD
Processing term 841-32-08 (eng)... 731-06‑03 MOD
Processing term 841-32-08 (fra)... 731-06‑03 MOD
Processing term 841-32-14 (eng)... 731-01‑09 MOD
Processing term 841-32-14 (fra)... 731-01‑09 MOD
Processing term 841-34-02 (eng)... 531-13‑06 MOD
Processing term 841-34-02 (fra)... 531-13‑06 MOD
Processing term 841-34-04 (eng)... 121-13‑16 MOD
Processing term 841-34-04 (fra)... 121-13‑16 MOD
Processing term 871-02-06 (fra)... [relaton-iso] ("ISO/IEC Guide 71:2014") fetching...
[relaton-iso] WARNING: no match found online for ISO/IEC Guide 71:2014. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO/IEC Guide 71:2014 (all parts)".
Processing term 871-04-13 (eng)... Recommendation ITU-T F.791:2015
Processing term 871-04-13 (fra)... Recommandation ITU-T F.791:2015
Processing term 904-01-04 (eng)... [relaton-iec] ("IEC 62430") fetching...
[relaton-iec] WARNING: no match found online for IEC 62430:2009. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 62430 (all parts)".
Processing term 904-01-15 (eng)... [relaton-iso] ("ISO 14001:2004") fetching...
[relaton-iso] WARNING: no match found online for ISO 14001:2004. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO 14001:2004 (all parts)".
Processing term 904-01-18 (eng)... [relaton-iso] ("ISO 14040:2006") fetching...
[relaton-iso] WARNING: no match found online for ISO 14040:2006. The code must be exactly like it is on the standards website.
[relaton-iso] If you wanted to cite all document parts for the reference, use "ISO 14040:2006 (all parts)".
Processing term 904-02-01 (eng)... [relaton-iec] ("IEC 62474") fetching...
[relaton-iec] WARNING: no match found online for IEC 62474:2012. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 62474 (all parts)".
Processing term 904-02-09 (eng)... [relaton-iec] ("IEC 62321") fetching...
[relaton-iec] WARNING: no match found online for IEC 62321:2008. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 62321 (all parts)".
Processing term 904-04-11 (eng)... [relaton-iec] ("IEC 62075") fetching...
[relaton-iec] WARNING: no match found online for IEC 62075:2012. The code must be exactly like it is on the standards website.
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 62075 (all parts)".
```

We should replace it with a normal empty space.

**Comment by skalee** [comment]
@ronaldtse There are several issues alike. Is it okay to fix things like that for ALL fields? I mean, descriptions too? I suppose yes, but I prefer to ask first just in case sometimes they are for purpose.

**Comment by ronaldtse** [comment]
@skalee I've done this particular fix in ed02caa, and yes we should do so. The problem is searchability and the expectations of users. The case of 00a0 is simple, but there might be some unicode issues that the correct solution is less obvious. In any case, these issues are to be reported back to IEC to decide on final treatment.

**Comment by ronaldtse** [comment]
This is also reported back to IEC.
